librz/bin/p/bin_xex.c

```diff
@@ -0,0 +1,75 @@
+// SPDX-FileCopyrightText: 2021 smac89 <noblechuk5[at]web[dot]de>
+// SPDX-License-Identifier: LPGL-3.0-only
+
+#include <rz_bin.h>
+#include <rz_lib.h>
+#include <rz_types.h>
+#include "librz/bin/format/xex/xex.h"
+
+/**
+ * \brief Check the buffer for the existence of the xex magic string
+ *
+ * \param b
+ */
+static bool check_buffer(RzBuffer *b) {
+    eprintf("[xex] check_buffer");
+	if (rz_buf_size(b) >= XEX_MAGIC_SIZE) {
+		ut8 buff[XEX_MAGIC_SIZE];
+		rz_buf_read_at(b, XEX_MAGIC_OFFSET, buff, XEX_MAGIC_SIZE);
+		return (!memcmp(buff, XEX_MAGIC, XEX_MAGIC_SIZE));
+	}
+	return false;
+}
+
+static bool load_buffer(RzBinFile *bf, void **bin_obj, RzBuffer *b, ut64 loadaddr, Sdb *sdb) {
+	eprintf("[xex] load_buffer");
```

Comment:
to be sure to have everything aligned, you can use clang-format:
```bash
cd /path/to/rizin
./sys/clang-format.py -f librz/bin/p/bin_xex.c
```
Be sure to have installed `gitpython` via `sudo pip install gitpython` and clang-format

Reply:
Thanks I just have to remember to run it before committing